src/callgrind.rs

```diff
@@ -4,38 +4,62 @@ use std::io;
 
 use initialize::StackFrame;
 
+// Stats about the relationship between two functions, one of which
+// calls the other.
 #[derive(Debug)]
 struct Call {
+    // How many times has the outer function called the inner one?
     count: usize,
+
+    // How many samples were found inside the inner function
+    // (including all sub-functions), when called by the outer?
```

Comment:
I don't understand the difference between the `count` and `inclusive` fields in this struct. Given 2 functions A and B, this is giving me information about calls from A to B. There are 2 possible ways I can think to measure calls from A to B given a bunch of stack traces:

1. count all direct calls A -> B
2. count all calls from A which eventually get to B somehow (possibly with multiplicity)

how do these 2 ways relate to the `count` and `inclusive` fields?